### ISSUE TITLE
There's no beta branch for native messenger

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#493]
Whatever we commit to master will be used by anyone using the native messenger.

It's kind of obvious but I was pretty surprised when I saw version 0.1.1 running on my laptop which is otherwise on stable.

Making this issue so that I am periodically reminded of this fact.

**Comment by glacambre** [comment]
We could maintain a lightweight tag named "stable" (created and updated with `git update-ref refs/tags/stable "$(git rev-list --tags -n 1)"`) and then download the native messenger from https://github.com/cmcaine/tridactyl/tree/stable/native/native_main.py with curl/wget/whatever.

**Comment by bovine3dom** [comment]
I feel like we might forget to update the tags, but it could be a good shout, since it doesn't really matter when we update the tags so long as it gets done eventually, unlike the releases.

**Comment by glacambre** [comment]
Maybe we could add a pre-push hook that detects if the "stable" tag isn't pointing to the last tag anymore and update it and automatically push it with `--no-verify`?

**Comment by glacambre** [comment]
Does the native messenger update itself when you don't build tridactyl? If not, should it? I think it'd be useful.

**Comment by glacambre** [comment]
I thought the native messenger wasn't automatically updated because I couldn't find the code that did that.
I believe it doesn't work. I restarted my browser multiple times and updated Tridactyl several times and I was still on version 1.0.3. I had to manually run install.sh in order to update to 1.0.5. Could the code to prevent Tridactyl to update while browsing be preventing the native messenger from being updated?
Perhaps onInstalled isn't triggered when Tridactyl is updated by restarting the browser?
